### ISSUE TITLE
Getting List Of Victims ....

### ISSUE THREAD

**Issue by vinhomesoceanpark** [Th30neAnd0nly/AIRAVAT#82]
![image](https://github.com/Th30neAnd0nly/AIRAVAT/assets/48808550/3878dc0a-af69-40b4-ab9f-b4b4e1c49c61)
Hi my web panel still is getting victim :( I dont know why. Can you help me pleaseee

**Comment by prateek-chaubey** [comment]
Seems like your database usage has been exhausted

**Comment by prateek-chaubey** [comment]
You can visit console.firebase.google.com and select your project to see the database usage

**Comment by vinhomesoceanpark** [comment]
![image](https://github.com/Th30neAnd0nly/AIRAVAT/assets/48808550/fe15586a-34fc-406f-9f33-3398ff1efa00)
This is report in database use. I dont know what happen :(

**Comment by vinhomesoceanpark** [comment]
![image](https://github.com/Th30neAnd0nly/AIRAVAT/assets/48808550/4bd80c1e-4efb-44d5-9cfe-04c1f87a3ff5)

**Comment by vinhomesoceanpark** [comment]
![image](https://github.com/Th30neAnd0nly/AIRAVAT/assets/48808550/c0b67258-f99d-49f0-90cc-1d51207e703c)
This is storage

**Comment by vinhomesoceanpark** [comment]
![image](https://github.com/Th30neAnd0nly/AIRAVAT/assets/48808550/cb8acf00-951e-4e2a-9746-54dcc6ba7310)
This is database

**Comment by vinhomesoceanpark** [comment]
![image](https://github.com/Th30neAnd0nly/AIRAVAT/assets/48808550/d1698d40-5b8e-4053-8626-c987f054713c)
this is overview :(

**Comment by prateek-chaubey** [comment]
Have you enabled the `Realtime Database` or `Cloud Firestore` ?

**Comment by vinhomesoceanpark** [comment]
![image](https://github.com/Th30neAnd0nly/AIRAVAT/assets/48808550/98c7666e-2605-4be9-9ce8-f79114063a04)
im use this menu sir

**Comment by vinhomesoceanpark** [comment]
i also trying with realtime database but also not working T.T

**Comment by prateek-chaubey** [comment]
You have to use `Realtime database` not `Firestore Database`

**Comment by jamilahmed2** [comment]
Hi
My realtime database is working fine but webpanel is not working like its stuck on processing.
Help plz


![Screenshot_20230822_191302_com android chrome](https://github.com/Th30neAnd0nly/AIRAVAT/assets/87638596/f5742c31-308d-4c11-a61e-3a1c5b9b4a18)
![Screenshot_20230822_115408](https://github.com/Th30neAnd0nly/AIRAVAT/assets/87638596/cc95dec7-2f28-4eab-8be1-0323bbd801b4)

**Comment by prateek-chaubey** [comment]
@jamilahmed2 You haven't hosted the files correctly, make sure to host them correctly on the webserver or on free hosting service providers

**Comment by jamilahmed2** [comment]
Is it mandatory to host a panel.

**Comment by prateek-chaubey** [comment]
Yes

**Comment by jamilahmed2** [comment]
Okai but i want more options like accessing camera live screening or stream the victms.

**Comment by Tanishqwerty** [comment]
I Hosted the files correctly on 000webhost and using realtime Database...And it is still not showing victims

**Comment by prateek-chaubey** [comment]
@Tanishqwerty Have you modified the rules?

**Comment by Tanishqwerty** [comment]
Yes i checked it was mistake on my end, its working now. Thanks For your response !!

**Comment by Tanishqwerty** [comment]
Can you help me with, I'm using AIRAVAT Pro, and it take Photos from back camera only, is there a way to turn it to front camera.
will Really Appreciate if you can help!!

**Comment by prateek-chaubey** [comment]
The AIRAVAT PRO you have been using isn't the authentic one and your personal data will get leaked online.

**Comment by Tanishqwerty** [comment]
You were right my google account and instagram got hacked, I recovered and secured my device now

**Comment by prateek-chaubey** [comment]
> You were right my google account and instagram got hacked, I recovered and secured my device now

See that is what i was talking about

**Comment by prateek-chaubey** [comment]
Telegram: @th30neand0nly
